osgar/drivers/rosmsg.py

```diff
@@ -639,6 +646,10 @@ def slot_raw(self, timestamp, data):
     def slot_desired_speed(self, timestamp, data):
         self.desired_speed, self.desired_angular_speed = data[0]/1000.0, math.radians(data[1]/100.0)
```

Comment:
In the 3d variant of the code, you are clearing the 2d command. Do you want to also clear the 3d command in this 2d variant?

Reply:
you are right, that it will be cleaner (i.e. not absolutely necessary but symmetry is better to understand). Fixed thanks